types/meteor/check.d.ts

```diff
@@ -1,24 +1,45 @@
 declare module "meteor/check" {
     module Match {
-        var Any: any;
-        var String: any;
-        var Integer: any;
-        var Boolean: any;
-        var undefined: any;
-        var Object: any;
```

Comment:
Most of these constants aren't actually defined on `Match`, so having them declared here is a bug.